src/main/webapp/reports/inventoryReports/grn_cash.xhtml

```diff
@@ -300,7 +300,7 @@
                                 </h:outputText>
                             </p:column>
                             <p:column headerText="Purchase Value" styleClass="text-end" headerStyleClass="text-end" width="11em">
-                                <h:outputText value="#{bi.billItemFinanceDetails.valueAtPurchaseRate}">
+                                <h:outputText value="#{bi.billItemFinanceDetails.purchaseRate * bi.billItemFinanceDetails.quantity}">
```

Comment:
Purchase Value in GRN should include the free quantities depending on the preference. It is should therefore NOT be calculated here.